doc/hcr.rst

```diff
@@ -26,59 +26,61 @@ To install SDL2 you can use ``nimble install sdl2``.
 .. code-block:: nim
 
   # logic.nim
-  import sdl2/sdl
-
+  import sdl2
+  
   #*** import the hotcodereloading stdlib module ***
   import hotcodereloading
-
+  
   var runGame*: bool = true
-  var window: Window
-  var renderer: Renderer
-
+  var window: WindowPtr
+  var renderer: RendererPtr
+  var evt = sdl2.defaultEvent
+  
   proc init*() =
-    discard sdl.init(INIT_EVERYTHING)
-    window = createWindow("testing", WINDOWPOS_UNDEFINED.cint, WINDOWPOS_UNDEFINED.cint, 640, 480, 0'u32)
-    assert(window != nil, $sdl.getError())
+    discard sdl2.init(INIT_EVERYTHING)
+    window = createWindow("testing", SDL_WINDOWPOS_UNDEFINED.cint, SDL_WINDOWPOS_UNDEFINED.cint, 640, 480, 0'u32)
+    assert(window != nil, $sdl2.getError())
     renderer = createRenderer(window, -1, RENDERER_SOFTWARE)
-    assert(renderer != nil, $sdl.getError())
-
+    assert(renderer != nil, $sdl2.getError())
+  
   proc destroy*() =
     destroyRenderer(renderer)
     destroyWindow(window)
-
-  var posX = 1
-  var posY = 0
-  var dX = 1
-  var dY = 1
-
+  
+  var posX : cint = 1
+  var posY : cint = 0
+  var dX : cint = 1
+  var dY : cint = 1
+  
   proc update*() =
-    for evt in events():
-      if evt.kind == QUIT:
+    while pollEvent(evt):
+      if evt.kind == QuitEvent:
         runGame = false
         break
-      if evt.kind == KEY_DOWN:
-        if evt.key.keysym.scancode == SCANCODE_ESCAPE: runGame = false
-        elif evt.key.keysym.scancode == SCANCODE_F9:
+      if evt.kind == KeyDown:
+        if evt.key.keysym.scancode == getScancodeFromName("Escape"): runGame = false
+        elif evt.key.keysym.scancode == getScancodeFromName("F9"):
```

Comment:
Looks like you can still use the constants, they have `SDL_` prefixes: https://github.com/nim-lang/sdl2/blob/46aa2c0e30e9edf6028bc432a3ed70e51a0fe9f9/src/sdl2/private/keycodes.nim#L4

Reply:
Thanks! I see them now. Commit `daccd8464b4186013ef5b59d13b3939512e2fdc1` resolved this.